### DISCUSSION TITLE
More Coupon Usage Limits: Exclude products, Limit by Taxonomy

### DISCUSSION THREAD

**Discussion by sbrow** [duncanmcclean/simple-commerce#1094] [Feature Requests]
Would you accept a PR that extends coupons with the following Usage Limits:

1. Blacklist products by id
2. Whitelist products by a configurable taxonomy
3. Blacklist products by a configurable taxonomy

Thanks!

**Comment by duncanmcclean** [comment]
I'd normally say yes but I'm currently not accepting PRs for large new features, since it'd conflict with some work I'm doing on a future version of Simple Commerce 😄 

I'll keep these requests in mind though for a future version.
